packages/core/src/html/NodeProcessor.js

```diff
@@ -57,6 +57,8 @@ class NodeProcessor {
     this.footnoteProcessor = new FootnoteProcessor();
     this.mdAttributeRenderer = new MdAttributeRenderer(this.markdownProcessor);
     this.pageNavProcessor = new PageNavProcessor();
+
+    this.processedModals = {};
```

Comment:
Double checking whether this will be handled correctly during live reloading?

Reply:
During live reload, a new instance of `NodeProcessor` should be created hence this should be updated accordingly for each of the regenerated pages.